strawberryfields/ops.py

```diff
@@ -1472,7 +1472,7 @@ def _decompose(self, reg, **kwargs):
 
                 if "symmetric" in mesh:
                     # Mach-Zehnder interferometers
-                    cmds.append(Command(MZgate(np.mod(phi, 2*np.pi), np.mod(theta, 2*np.pi)), (reg[n], reg[m])))
+                    cmds.append(Command(MZgate(np.mod(theta, 2*np.pi), np.mod(phi, 2*np.pi)), (reg[n], reg[m])))
```

Comment:
Another one we missed 🙂

Reply:
Did you search for MZgate in all files to find this one?

Reply:
No, it revealed itself as I added the new tests. Goes to show the important of having proper tests with good coverage!

Reply:
I just ran a full text search, and did not find any additional issues withing SF.

Reply:
However, I realized there is now a problem in line 60+61 of circuitspecs/chip2.py (as now even and odd indices should be swapped in below docstring which is the "current, outdated" version):
```
        # even phase indices correspond to external Mach-Zehnder interferometer phases
        # odd phase indices correspond to internal Mach-Zehnder interferometer phases
```
which propagates all the way to chipdata and even our interferometer calibration routines, namely the phase array convention.

Reply:
So I guess the best, least error-prone fix would be to keep the phase indices in the templates as they are, just change the docstrings, and implement these changes in github.com/xanaduai/interferometerCalibration.

Reply:
Summarizing a conversation @lneuhaus and I had offline, the solutions here are:

* Keep the changes in this PR, but modify the templates in the rest of the stack so that internal phase comes first in MZgates. Pros: having internal phase come first is a better convention, matches the BSgate. Cons: more work.

* Revert the changes in this PR, and instead keep the MZgate using the convention where external phase comes first.

Reply:
@josh146 @lneuhaus which of the two solutions would you recommend?

Reply:
The first one, half way done already.